### PR TITLE
[IMP] crm_claim_mrp: Change the one to many to many to many in claims…

### PR DESCRIPTION
**Pull request by Berezi**
… to production and workorder.

### CODE REVIEW

**Comment by Berezi** [comment]
@anajuaristi @oihane @Daniel-CA @alfredoavanzosc review please.

File: crm_claim_mrp/models/crm_claim.py [hunk 1]
```diff
@@ -27,3 +27,15 @@ def onchange_workorder_id(self):
...
                 'production_id', '=', False)]
         return {'domain': {'workorder_id': domain}}
+
+    @api.onchange('workorder_id', 'production_id', 'date',
+                  'model_ref_id', 'workorder_id.workcenter_id', )
+    def onchange_name(self):
+        if self.model_ref_id and self.workorder_id and (
+            self.workorder_id.workcenter_id) and (
+                self.production_id):
+            self.name = False
```
Comment (alfredoavanzosc) [inline crm_claim_mrp/models/crm_claim.py]: Esta linea está de sobra.

Comment (Berezi) [reply crm_claim_mrp/models/crm_claim.py]: Aldaketa eginda. Eskerrik asko.

**Review by alfredoavanzosc — APPROVED** [review]
:+1:

**Review by anajuaristi — APPROVED** [review]

**Pull request merged**